static/src/js/util/createSpatialDisplay.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Returns a string displaying the spatial information.
+ * @param {Object} spatial Object that holds the different spatial areas.
+ * @returns {String} Returns a string formatting the spatial areas into human readable values.
+ */
+export const createSpatialDisplay = (spatial) => {
+  const {
+    boundingBox,
+    circle,
+    line,
+    point,
+    polygon
+  } = spatial
```

Comment:
I _think_ this should work if the user uploaded a shapefile, because we pull the spatial out of the geojson and use it to send to CMR, but we need to verify that it does work since `supportsShapefileSubsetting` is one of the triggers for seeing that `enableSpatialSubsetting` checkbox

Reply:
@macrouch Could I get some guidance as to how to pull out a spatial from the geojson?

Reply:
I hope you don't have to. I'd first verify if you see the enableSpatialSubsetting checkbox with a shapefile added. If everything still works then it should be good to go

Reply:
How would I add a shapefile? Right now I test it with the polygon attribute set which for some reason activates the supportsShapefileSubsetting

Reply:
On the search page instead of adding spatial directly drop a shapefile onto the map, github won't let me upload one here, check slack

Reply:
Looks good.